### PR TITLE
Filename fix + dtype warnings

### PR DESCRIPTION
**Pull request by ackagel**
The is a patch for the file naming issue addressed in issue #135.  In particular, this PR:

 - reorganizes/renames some files+directories in the input_data folder
 - adds fov name delimiter auto-detection to the mibi+multi tiff loading function
 - changes a ValueError into a warning for int-expected float image loading
 - alters notebooks accordingly
 - will add more testing to cover new functionality

### CODE REVIEW

File: segmentation/utils/data_utils_test.py [hunk 1]
```diff
@@ -14,9 +15,23 @@
 
 importlib.reload(data_utils)
 
+# required metadata for mibitiff writing (barf)
```
Comment (ngreenwald) [inline segmentation/utils/data_utils_test.py:18]: 👍

File: segmentation/utils/data_utils_test.py [hunk 2]
```diff
@@ -104,6 +120,49 @@ def test_load_imgs_from_mibitiff():
         data_xr.values[0],
         (tiff.read(os.path.join(data_dir, mibitiff_files[0])))[channels].data)
 
+    # this is looped to make sure temp_file doesn't have '_' in its name
+    # (sadly there's no excluded characters option for tempfile)
+    while True:
+        with tempfile.NamedTemporaryFile(suffix=".tif", dir=data_dir) as temp_file:
```
Comment (ngreenwald) [inline segmentation/utils/data_utils_test.py:126]: I did some googling, you're right. 

What about renaming it instead? os.rename

Comment (ackagel) [reply segmentation/utils/data_utils_test.py:126]: I could try it, but I think I would have to handle cleanup manually then.  Alternatively, I could create a temporary directory, and create two mibi tiffs (or copy one in input_data) to test in it, and then just delete the directory.  Now that I type it out, that seems better than literally gambling on no underscores, so I'll probably do that.

**Comment by ackagel** [comment]
yeah I'm not sure.  Something got messed up since I had an editor open while pulling changes from master, and I haven't been able to get rid of those commits for some reason.  Thankfully, it shouldn't hurt anything and they should go away on my end once this PR is merged into master.

**Pull request merged**